### PR TITLE
Fix : Change master branch to main

### PR DESCRIPTION
**Pull request by loprima-l**
This fix should change the edit branch to main in the documentation

I didn't update the CHANGELOG, as it doesn't concern the app itself. Let me know if I should do this 

- [x] read the [CONTRIBUTING guidelines](README.md#contributing)
- [ ] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [ ] updated CHANGELOG.md

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 site_name: Headscale
 site_url: https://juanfont.github.io/headscale
+edit_uri: blob/main/docs/ #Change the master branch to main as we are using main as a main branch
```
Comment (juanfont) [inline mkdocs.yml:3]:
```suggestion
edit_uri: blob/main/docs/  # Change the master branch to main as we are using main as a main branch
```

**Comment by loprima-l** [comment]
Fixed !

File: mkdocs.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 site_name: Headscale
 site_url: https://juanfont.github.io/headscale
+edit_uri: blob/main/docs/  # Change the master branch to main as we are using main as a main branch
```
Comment (juanfont) [inline mkdocs.yml:3]:
```suggestion
edit_uri: blob/main/docs/ # Change the master branch to main as we are using main as a main branch
```

**Review by juanfont — APPROVED** [review]

**Comment by juanfont** [comment]
Thanks!

**Pull request merged**